scripts/postlink.js

```diff
@@ -383,7 +386,8 @@ function addSentryInit() {
 function resolveSentryCliBinaryPath(props) {
   return new Promise(function(resolve, reject) {
     try {
-      props['cli/executable'] = require.resolve('sentry-cli-binary/bin/sentry-cli');
+      const cliPath = require.resolve('sentry-cli-binary/bin/sentry-cli');
+      props['cli/executable'] = path.relative(process.cwd(), cliPath);
```

Comment:
We actually don't want to have an absolute path because when we write this in the properties file and when the user check this in to their CI we will get a bad path because `/User/name/bla/bla` isn't valid on the CI.
And we need to resolve the sentry-cli-binary with `require.resolve` because it could be in `node_modules` directly or within `react-native-sentry/node_modules` depending on the npm version the user is using.
So by converting the absolute path to a relative path this should always work.